rtcp.c

```diff
@@ -133,6 +133,47 @@ static void janus_rtcp_incoming_sr(janus_rtcp_context *ctx, janus_rtcp_sr *sr) {
 	ctx->lsr = (ntp >> 16);
 }
 
+/* Link quality estimate filter coefficient */
+#define LINK_QUALITY_FILTER_K 3.0
+
+static double janus_rtcp_link_quality_filter(double last, double in) {
+	if (last == 0) {
+		return in;
+	}
+	else {
+		return (1.0 - 1.0/LINK_QUALITY_FILTER_K) * last + (1.0/LINK_QUALITY_FILTER_K) * in;
+	}
+}
+
+/* Update link quality stats based on RR */
+static void janus_rtcp_rr_update_stats(rtcp_context *ctx, report_block rb) {
```

Comment:
This should be `janus_report_block`, as we recently typedeffed all the RTCP structs to have the `janus_` prefix: the version without may be deprecated and removed in the future.